explainaboard/tests/test_aspect_based_sentiment_classification.py

```diff
@@ -12,7 +12,7 @@ class TestAspectBasedSentimentClassification(unittest.TestCase):
     def test_e2e(self):
         metadata = {
             "task_name": TaskType.aspect_based_sentiment_classification.value,
-            "metric_names": ["Accuracy", "F1score"],
+            "metric_names": ["Accuracy", "F1Score"],
```

Comment:
do we finally aim to replace this with metric's `default_name`?

Reply:
I think thinking about this would be good, but not sure if it needs to be done now.